gnomad/utils/annotations.py

```diff
@@ -2326,28 +2339,78 @@ def _create_group_dicts(
         "cohort": {"id": "ALL"},
     }
 
-    # Create ancillaryResults for additional frequency and popMaxFAF95 information
+    # Create ancillaryResults for additional frequency and popMaxFAF95 information.
     ancillaryResults = {
         "homozygotes": overall_freq["homozygote_count"],
-        "popMaxFAF95": {
-            "frequency": input_struct.faf95.popmax,
-            "confidenceInterval": 0.95,
-        },
     }
 
-    if input_struct.faf95.popmax_population is not None:
-        ancillaryResults["popMaxFAF95"][
-            "popFreqID"
-        ] = f"{gnomad_id}.{input_struct.faf95.popmax_population.upper()}"
-    else:
-        ancillaryResults["popMaxFAF95"]["popFreqID"] = None
+    # Add hemizygote count if not autosomal or PAR.
+    if not input_struct.in_autosome_or_par:
+        hemizygote_count = input_struct.freq[freq_index_dict["XY_adj"]].AC
+        ancillaryResults["hemizygotes"] = hemizygote_count
```

Comment:
is this the same code that starts at line #2279? This looks like the simpler version here, but I wanna make sure that I'm not just turned around

Reply:
Essentially yes, the reason this is simpler is because it's the top level group so the key is always just `XY_adj`. In the inner function it needs to check whether the current group is sexed or not, and if not, query the `freq` for the sexed sub-group.

Reply:
Can we remove the code at line #2279 then? Again sorry if I'm parsing this wrong

Reply:
adding a new review, pinging this again if we can remove the code at line #2279-2290 given this code

Reply:
@matren395 I don't think the lines 2279-2289 can be removed or combined here, it adds logic to look up the specific population group. It's possible both 2348-2350 and 2279-2289 could be pulled out into a function but it would probably be more lines and more complicated

Reply:
noted! thanks for the explanation